articles/java/sdk/java-sdk-azure-get-started.md

```diff
@@ -248,15 +225,22 @@ Replace the main method in `AzureApp.java` with the one below, updating the `app
     public static void main(String[] args) {
         try {
 
-            final File credFile = new File(System.getenv("AZURE_AUTH_LOCATION"));
             final String appName = "YOUR_APP_NAME";
 
-            Azure azure = Azure.configure()
-                    .withLogLevel(LogLevel.BASIC)
-                    .authenticate(credFile)
+            TokenCredential credential = new EnvironmentCredentialBuilder()
+                    .authorityHost(AzureAuthorityHosts.AZURE_PUBLIC_CLOUD)
+                    .build();
+
+            // if you do not set tenant ID and subscription ID via environment variables
+            // change to create azure profile with tenantId, subscriptionId and azure environment
+            AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+            
+            AzureResourceManager azureResourceManager = AzureResourceManager.configure()
+                    .withLogLevel(HttpLogDetailLevel.BASIC.BASIC)
```

Comment:
what's this? make sure the code at least compiles.

Reply:
It's copy-paste issue here. As no compile error would throw, I just missed it. Fixed now.